packages/eslint-plugin/src/rules/key-spacing.ts

```diff
@@ -14,6 +14,16 @@ const baseSchema = Array.isArray(baseRule.meta.schema)
   ? baseRule.meta.schema[0]
   : baseRule.meta.schema;
 
+/**
+ * To replace with native .at() once Node 14 stops being supported
```

Comment:
Nit: we generally use actual `TODO` comments to help with searchability.

```suggestion
 * TODO: replace with native .at() once Node 14 stops being supported
```